xml/ha_config_cli.xml

```diff
@@ -356,9 +387,187 @@ Operations' defaults (advisory minimum):
    </tip>
   </sect2>
 
+  <sect2 xml:id="sec.ha.manual_config.clusterscripts">
+    <title>Using &crmsh;'s Cluster Scripts</title>
+    <remark>toms 2015-10-14: FATE#318211</remark>
+    <para>One big task of configuring and managing a cluster is to collect
```

Comment:
Perhaps shorten this para to: Collecting information from all cluster nodes and deploying any changes is a key cluster administration task. Instead of performing the same procedures manually on different nodes (which is error-prone), you can use the &crmsh; cluster scripts.